src/lib/api/group/client/groupApiClient.client.ts

```diff
@@ -77,3 +77,34 @@ export const putRenameGroup = async (
     return Err('API response is invalid');
   }
 };
+
+/**
+ * Interface representing the API response for the function to generate a invitation link
+ * @property invitationLinkHash - string, a hash for invitation link
+ */
+export interface IGenerateInvitationLink {
+  invitationLinkHash: string;
+}
+
+/**
+ * generate a invitation link hash
+ * @param groupId unique Id for generating invitation link hash
+ * @returns A hash: IGenerateInvitationLink of inivitation link
+ */
+export const generateInvitationLinkHash = async (
```

Comment:
As per convention, please rename it to start with `put` (one of the request methods).